### DISCUSSION TITLE
SSH to Remote Linux Machine through multiple Windows Jump Hosts

### DISCUSSION THREAD

**Discussion by elang-velusamy** [Eugeny/tabby#9034] [Q&A] [answered]
Currently, to connect to my Linux machine, i have to connect to a Windows Jump Host (WJH-1) using rdp. Direct SSH connection is not enabled in this Windows JH. However, I have another Windows JH (WJH-2) in which SSH is enabled though which i can take SSH connection to WJH-1.

Now, is it possible to connect to the Linux machine though the Jump Hosts WJH-2 and WJH-1? How to set it up? Please help.

**Comment by Eugeny** [comment, marked as answer]
You can chain jumphosts by setting the respective "previous" jumphost for each connection - Tabby will connect them with each other correctly

**Reply by elang-velusamy** [reply]
Thanks Eugeny, it worked! 👍
